cli/cmd/dash.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2022 CFC4N <cfc4n.cs@gmail.com>. All Rights Reserved.
```

Comment:
Please disable compilation of the Android version, including other golang files involving `dash/zsh`.

```go
//go:build !androidgki
// +build !androidgki
```